Isabelle/Chi_Calculus_Examples/Diamond.thy

```diff
@@ -194,7 +194,8 @@ proof -
     sorry
   then have "a \<triangleright>\<^sup>\<infinity> x. \<Prod> c\<leftarrow>[b]. c \<triangleleft> x \<approx>\<^sub>\<flat> a \<triangleright>\<^sup>\<infinity> x. b \<triangleleft> x"
     by (simp add: basic_weak_multi_receive_preservation)
-  then show ?thesis .
+  then show ?thesis
+    by (simp add: distributor_def unidirectional_bridge_def)
```

Comment:
Here, the use of `unfolding` would make it unnecessary to invoke the `simp` method, allowing us to use the trivial proof `.`.